common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExternalBlockStoreClient.java

```diff
@@ -43,6 +48,9 @@
  * (via BlockTransferService), which has the downside of losing the data if we lose the executors.
  */
 public class ExternalBlockStoreClient extends BlockStoreClient {
+  private static final Logger logger = LoggerFactory.getLogger(ExternalBlockStoreClient.class);
```

Comment:
The abstract class `BlockStoreClient` already initialized a `logger`. Why need a new one?

Reply:
This is perhaps a miss while we forward port the patch from our internal Spark 2.3 branch to master.
In Spark 2.3, the logger was only created for then class `ExternalShuffleClient`.
This has already changed as you mentioned.
Will remove it.